sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -909,8 +909,8 @@ typedef struct CommandListBatchConfig {
 
   // Thresholds for when increase batch size (number of closed early is small
   // and number of closed full is high).
-  pi_uint32 NumTimesClosedEarlyThreshold{2};
-  pi_uint32 NumTimesClosedFullThreshold{10};
+  pi_uint32 NumTimesClosedEarlyThreshold{3};
+  pi_uint32 NumTimesClosedFullThreshold{8};
```

Comment:
I think we should add controls for users to revert to old behavior in case they get a U2U regressions (or use other values that work best for their case).